src/App.jsx

```diff
@@ -1,107 +1,25 @@
+/* eslint-disable max-len */
```

Comment:
Try not to disable eslint rules unless necessary. If 'max-len' rule is causing problems, you may want to consider breaking your code into smaller, more manageable pieces.